libcomps/src/python/tests/__test.py

```diff
@@ -737,6 +737,22 @@ def test_xml(self):
         comps4 = libcomps.Comps()
         self.assertRaises(libcomps.ParserError, comps4.fromxml_str, str(INVALID_COMPS_XML))
 
+        VALID_XML_NO_COMPS = "<whatever />"
+        comps5 = libcomps.Comps()
+        ret = comps5.fromxml_str(str(VALID_XML_NO_COMPS))
+        # return code 1 is non fatal error
+        self.assertTrue(ret == 1, comps5.get_last_errors())
+
+        VALID_XML_NO_COMPS = """<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Minor: maybe just don't use the same variable name `VALID_XML_NO_COMPS` for two different tests, it's kinda confusing...

Reply:
Sure, updated!